.github/workflows/run_rc_validation_go_wordcount.yml

```diff
@@ -129,6 +131,7 @@ jobs:
             --project=${{ env.GCP_PROJECT_ID }} \
             --region=${{ env.GCP_REGION }} \
             --temp_location=${{ env.GCS_TEMP_LOCATION }} \
+            --staging_location=${{ env.GCS_STAGING_LOCATION }} \
```

Comment:
Similarly, this does not submit the job with an rc version of Beam Go, it just uses the RC container.